sim/battle-actions.ts

```diff
@@ -1,6 +1,6 @@
 import { Dex, toID } from './dex';
 
-const CHOOSABLE_TARGETS = new Set(['normal', 'any', 'adjacentAlly', 'adjacentAllyOrSelf', 'adjacentFoe']);
+const CHOOSABLE_TARGETS = new Set(['normal', 'any', 'adjacentAlly', 'adjacentAllyOrSelf', 'adjacentFoe','randomNormal']);
```

Comment:
This _can't_ be correct code...

Reply:
Like even ignoring the formatting issue... `randomNormal` moves (I think that's, what, stuff like Outrage?) shouldn't ever have choosable targets?